packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -31,14 +31,17 @@ export const sendBoxStyle = mergeStyles({
  * @private
  */
 export const sendButtonStyle = mergeStyles({
-  margin: '.6rem'
+  marginTop: '0.567rem',
+  marginBottom: '0.559rem',
```

Comment:
nit: these values look super specific and hence become very fragile. Is there something we can do that makes it aligned vertically automatically? (I've found often setting lineHeight to be the same as the height to work for text so might work here, or maybe wrap the icon in a stack that can vertically align its children? I've never found a consistent way to make it vertically centered so may need some random trial and error with css props)